x-pack/plugins/remote_clusters/public/sections/remote_cluster_list/components/disconnect_button/disconnect_button.js

```diff
@@ -135,7 +136,10 @@ export class DisconnectButtonUi extends Component {
 
     return (
       <Fragment>
-        <EuiButton color="danger" onClick={this.showConfirmModal}>
+        <EuiButton
+          color="danger"
+          onClick={this.showConfirmModal}
+        >
           {this.renderButtonText()}
```

Comment:
I am not sure about the "disconnect" naming. What we are really doing is adding and removing clusters. Those can have a connected status or not but there is no way that a user can connect or disconnect a cluster. See in the screenshot, it is weird to offer to disconnect a non connected cluster 😊 

<img width="966" alt="screen shot 2018-11-28 at 12 17 48" src="https://user-images.githubusercontent.com/2854616/49148448-b57d2b80-f307-11e8-969f-c2c7a2aff6ac.png">

Reply:
You're totally right, I'll rename everything to use "Add" and "Remove" terms.